modules/microshift-data-cleaning-container-images.adoc

```diff
@@ -0,0 +1,86 @@
+// Module included in the following assemblies:
+//
+// * microshift_troubleshooting/microshift-cleanup-data.adoc
+
+:_mod-docs-content-type: PROCEDURE
+[id="microshift-data-cleaning-container-images_{context}"]
+= Cleaning all data and keeping the container images
+
+Retain the {microshift-short} container images while cleaning all data by running the `microshift-cleanup-data` script with the `--all` and `--keep-images` arguments.
+
+Keeping the container images helps speed up {microshift-short} restart after data cleanup because the necessary container images are already present locally when you start the service.
+
+When you run the script with the `--all` and `--keep-images` arguments, you perform the following cleanup actions:
+
+* All {microshift-short} services are stopped and disabled
+* Deletes all {microshift-short} pods
+* Resets network configuration
+* Deletes the `/var/lib/microshift` data directory
+* Deletes OVN-K networking configuration
+
+[WARNING]
+====
+This option deletes all {microshift-short} data and user workloads. Use with caution.
+====
+
+.Prerequisites
+. You are logged into the {microshift-short} as an administrator with root-user access.
+. You have filed a support case.
+
+.Procedure
+
+. Cleanup all data and user workloads while retaining the {microshift-short} container images by running the `microshift-cleanup-data` script with the `--all` and `--keep-images` argument, by entering the following command:
```

Comment:
I think since you say "by running the following command", and since the source block below already states what is part of the command, you can remove this part of the step:
```suggestion
. Cleanup all data and user workloads while retaining the {microshift-short} container images by entering the following command:
```